src/hotspot/share/services/attachListener.cpp

```diff
@@ -183,17 +183,13 @@ static jint thread_dump(AttachOperation* op, outputStream* out) {
     }
   }
 
-  // thread stacks
-  VM_PrintThreads op1(out, print_concurrent_locks, print_extended_info);
+  // thread stacks and JNI global handles
+  VM_PrintThreads op1(out, print_concurrent_locks, print_extended_info, true);
```

Comment:
Please comment what the true argument means.

Reply:
fixed